eng/IndirectReferences.props

```diff
@@ -0,0 +1,1570 @@
+		<IndirectReference Include="Microsoft.Net.Http.Headers" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Http" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Mvc.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Hosting.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Authentication.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.ResponseCaching.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Http.Extensions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Routing" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Authorization.Policy" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Metadata" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Routing.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Mvc.Core" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Http.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Hosting.Server.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Http.Features" KeepDuplicates="false" />
+	</ItemGroup>
+	<ItemGroup Condition="@(Reference->Count()) != '0' AND @(Reference->AnyHaveMetadataValue('Identity', 'Microsoft.AspNetCore.Authentication'))">
+		<IndirectReference Include="Microsoft.AspNetCore.WebUtilities" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.DataProtection" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Authentication.Core" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.Net.Http.Headers" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Http" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Cryptography.Internal" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.DataProtection.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Http.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Http.Features" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Authentication.Abstractions" KeepDuplicates="false" />
+		<IndirectReference Include="Microsoft.AspNetCore.Http.Extensions" KeepDuplicates="false" />
+	</ItemGroup>
+	<ItemGroup Condition="@(Reference->Count()) != '0' AND @(Reference->AnyHaveMetadataValue('Identity', 'Microsoft.AspNetCore.Html.Abstractions'))">
+	</ItemGroup>
+	<ItemGroup Condition="@(Reference->Count()) != '0' AND @(Reference->AnyHaveMetadataValue('Identity', 'Microsoft.Net.Http.Headers'))">
```

Comment:
This doesn't look right. This has a reference to Microsoft.Extensions.Primitives.